.changeset/beige-elephants-perform.md

```diff
@@ -0,0 +1,8 @@
+---
+'@sl-design-system/message-dialog': patch
+---
+
+Various improvements:
+- Add `autofocus` attribute to `MessageDialogButton` config
+- Per the documentation, cancel buttons must use `fill="outline"` and `variant="primary"`
+- Improve API documentation
```

Comment:
Maybe we want to also add in the `message-dialog/code.md` file 
`{% include "../component-table.njk" %}` for the API table like in other components?

Reply:
Done